bodyfetcher.py

```diff
@@ -208,10 +241,37 @@ def make_api_call_for_site(self, site):
                 owner_rep = 0
             q_id = str(post["question_id"])
 
-            is_spam, reason, why = check_if_spam(title, body, owner_name, owner_link, site, q_id, False, False, owner_rep, post_score)
+            is_spam, reason, why = check_if_spam(
```

Comment:
@hichris1234 I was considering the idea of creating temporary model files before they get passed into functions so we can fill the models when they come in and unpack them when they get out to avoid confusion (unknown property usage vs named properties is much more clear) and it would simplify the massive amount of parameters going into each function